### PR TITLE
OSDOCS#11004: Hosted control planes: Migrate AWS to OCP

### PR DESCRIPTION
**Pull request by xenolinux**
This PR is a part of HCP content migration and modularization. The content has not been changed for this PR. 
This content is only being moved from RHACM to OCP. This content has been SME and peer reviewed earlier. This migration PR contains the QE approval

Version(s): 4.17 and main

Issue: https://issues.redhat.com/browse/OSDOCS-11004

Link to docs preview:  
- Whole assembly -- https://81797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-aws.html
- https://81797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-bm#hcp-bm-hc_hcp-deploy-bm -- Please see Additional resources
- https://81797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-non-bm#hcp-non-bm-hc_hcp-deploy-non-bm -- Please see Additional resources
- https://81797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-import
- https://81797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-enable-disable#hcp-disable_hcp-enable-disable

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-bm-access.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can access the hosted cluster by either getting the `kubeconfig` file and `k
 
 .Prerequisites
 
-To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted control plane clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
+To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
```
Comment (jparrill) [inline modules/hcp-bm-access.adoc:14]: "The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources"

The hosted cluster (hosting) namespace contains the hosted cluster resources, including the access secrets.

File: modules/hcp-virt-access.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can access the hosted cluster by either getting the `kubeconfig` file and `k
 
 .Prerequisites
 
-To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted control plane clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
+To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
```
Comment (jparrill) [inline modules/hcp-virt-access.adoc:13]: "The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and "

The hosted cluster (hosting) namespace contains the hosted cluster resources, including the access secrets. The _hosted control plane_ namespace is where the hosted control plane runs.

**Review by jparrill — COMMENTED** [review]
Dropped some comments. Thanks!

File: modules/hcp-aws-enable-ext-dns.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+External DNS creates name records for hosted cluster `Services` that specify a publishing type of `LoadBalancer` or `Route` and provide a hostname for that publishing type. For hosted clusters with `Private` or `PublicAndPrivate` endpoint access types, only the `APIServer` and `OAuth` services support hostnames. For `Private` hosted clusters, the DNS record resolves to a private IP address of a Virtual Private Cloud (VPC) endpoint in your VPC.
+
+A hosted control plane exposes the following services:
+
+* `APIServer`
+* `OAuthServer`
+* `Konnectivity`
+* `Ignition`
+* `OVNSbDb`
```
[the remark this answers is not in the document]
Comment (csrwng) [reply modules/hcp-aws-enable-ext-dns.adoc:25]: Yes, it's no longer needed

[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/hcp-aws-enable-ext-dns.adoc]: Addressed

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM! Nice work!

**Comment by xenolinux** [comment]
Addressed all peer review comments.

**Pull request merged**